packages/client/components/ActivityLibrary/CreateNewActivity/CreateNewActivity.tsx

```diff
@@ -95,9 +95,8 @@ const SUPPORTED_CUSTOM_ACTIVITIES: SupportedActivity[] = [
 const query = graphql`
   query CreateNewActivityQuery {
     viewer {
-      featureFlags {
-        noTemplateLimit
-      }
+      freeCustomRetroTemplatesRemaining
```

Comment:
I created two flags, `freeCustomRetroTemplatesRemaining` and `freeCustomPokerTemplatesRemaining`, thinking of our engineering principle that we should repeat the code twice and, on the third time, refactor it into something reusable.